pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -170,7 +170,7 @@ func (c *client) RunTask(ctx context.Context, taskDefinition types.TaskDefinitio
 
 	_, err := c.ecsClient.RunTask(ctx, input)
 	if err != nil {
-		return fmt.Errorf("failed to run ECS task %s: %w", *taskDefinition.TaskDefinitionArn, err)
+		return fmt.Errorf("failed to run ECS task %s: %w", *taskDefinition.Family, err)
```

Comment:
Should keep it as is and add a check for TaskDefinicationArn before running task since it doesn't make sense to run a task that does not exist. Please refer to 
https://github.com/pipe-cd/pipecd/blob/21936efa898995e4cdaefb91710f956ae1fe0d52/pkg/app/piped/platformprovider/ecs/client.go#L179-L181